### PR TITLE
Issues templates

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Prepare issues templates.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Nothing to test, this file can be editing easily and don't affect the core.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9424)

### CODE REVIEW

**Comment by kpodemski** [comment]
NO WAY, are you enabling Issues on Github?! No more JIRA?!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Why is there a space here? 😄

**Comment by eternoendless** [comment]
@kpodemski I can't tell if you're happy or annoyed about it 😅...

But the answer is YES we are going to drop JIRA and go full GitHub! 💪

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (PierreRambaud) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: Was generated by github :P

**Comment by kpodemski** [comment]
@eternoendless oh, that was something what we wanted **years** ago, finally, no more JIRA!

super news, really, well done guys, well done

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**